wasmcloud_basics/wash_intro/start.md

```diff
@@ -0,0 +1,36 @@
+## Starting wash
+
+`wash` can be run in two modes, interactive or single commands. In this tutorial we will explore the full power of `wash` in interactive mode. In order to start the `wash` REPL, we run `wash up`
```

Comment:
I think this first sentence may read better as "wash is a command line interface (CLI) that also supports a fully interactive REPL mode". That way we set the stage that this can all be done on the command line